ray-operator/config/samples/ray-data-image-resize/ray-data-image-resize-gcsfusecsi-job.yaml

```diff
@@ -0,0 +1,198 @@
+              name: ray-logs
+            - name: dshm
+              mountPath: /dev/shm
+            - mountPath: /data
+              name: gcs-fuse-csi-ephemeral
+            - mountPath: /home/ray/code-samples
+              name: code-samples
+          volumes:
+          - emptyDir: {}
+            name: ray-logs
+          - name: dshm
+            emptyDir:
+              medium: Memory
+          - csi:
+              driver: gcsfuse.csi.storage.gke.io
+              volumeAttributes:
+                bucketName: gcp-public-data-landsat
+                mountOptions: implicit-dirs,anonymous-access,uid=1000,gid=100,metadata-cache:ttl-secs:-1,metadata-cache:stat-cache-max-size-mb:-1,metadata-cache:type-cache-max-size-mb:-1
+                skipCSIBucketAccessCheck: 'true'
+            name: gcs-fuse-csi-ephemeral
+          - configMap:
+              items:
+              - key: ray_data.py
+                path: ray_data.py
+              name: code-samples
+            name: code-samples
+    rayVersion: 2.9.3
+    workerGroupSpecs:
+    - groupName: worker-group
+      maxReplicas: 3
+      minReplicas: 1
+      rayStartParams: {}
+      replicas: 3
+      template:
+        metadata:
+          annotations:
+            gke-gcsfuse/cpu-limit: '0'
+            gke-gcsfuse/ephemeral-storage-limit: '0'
+            gke-gcsfuse/memory-limit: '0'
+            gke-gcsfuse/volumes: 'true'
+        spec:
+          containers:
+          - image: rayproject/ray:2.9.3
+            name: ray-worker
+            resources:
+              requests:
+                cpu: '1'
+                memory: 4Gi
+            volumeMounts:
+            - mountPath: /tmp/ray
+              name: ray-logs
+            - name: dshm
+              mountPath: /dev/shm
+            - mountPath: /data
+              name: gcs-fuse-csi-ephemeral
+            - mountPath: /home/ray/code-samples
+              name: code-samples
+          volumes:
+          - emptyDir: {}
+            name: ray-logs
+          - name: dshm
+            emptyDir:
+              medium: Memory
+          - csi:
+              driver: gcsfuse.csi.storage.gke.io
+              volumeAttributes:
+                bucketName: gcp-public-data-landsat
+                mountOptions: implicit-dirs,anonymous-access,uid=1000,gid=100,metadata-cache:ttl-secs:-1,metadata-cache:stat-cache-max-size-mb:-1,metadata-cache:type-cache-max-size-mb:-1
+                skipCSIBucketAccessCheck: 'true'
+            name: gcs-fuse-csi-ephemeral
+          - configMap:
+              items:
+              - key: ray_data.py
+                path: ray_data.py
+              name: code-samples
+            name: code-samples
+  runtimeEnvYAML: |
+    pip:
+      - torch
+      - torchvision
+      - numpy
+    env_vars:
+      BUCKET_PREFIX: "LC08/01/001/002"
+  shutdownAfterJobFinishes: true
+  ttlSecondsAfterFinished: 30
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: code-samples
```

Comment:
Put the source code in the repo and reference it as a runtime environment, see the existing example https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-data-image-resize/ray-job.image-resize.yaml#L14

Reply:
done